doc/Analytics.xml

```diff
@@ -3751,8 +3751,9 @@ xmlns:acme="http://www.acme.com/schema"&gt;
     <section>
       <title>Events</title>
       <section>
-        <title>Audio Detected</title>
-        <para>Analytics engine can send the following event for analyzed audio data.</para>
+        <title>Audio Detected (deprecated)</title>
+        <para>Analytics engine can send the following event for analyzed audio data. This event is
+          deprecated in favor events described in <xref linkend="appendix_hw3_vnr_nwb"/>.</para>
```

Comment:
Minor typo,  **deprecated in favor of events described in** ...

Reply:
Fixed